content/cybersecurity/concepts/nmap/terms/tcp-syn-scan/tcp-syn-scan.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'TCP SYN Scan'
+Description: 'Performs a scan quickly, efficientily and stealthly. The most popular scan option'
```

Comment:
Can we rephrase here? The most popular scan option depends on a user's specific needs so I think we can remove the second sentence. In addition, can we include in the description what it scans and its purpose?

Reply:
I thought about changing the description to this:

Performs a scan that can indicate open, filtered and closed port states with a reduced risk of detection, since it never completes a full TCP connection

What do you think?